suites/squid/nvmeof/tier-2_8-1_nvmeof_gateway_operations.yaml

```diff
@@ -0,0 +1,713 @@
+              args:
+                subsystem: nqn.2016-06.io.spdk:cnode1.gw-group1
+                nsid: 1
+                rw-ios-per-second: 20
+              validate:
+                omap: true
+          - config:
+              service: namespace
+              command: resize              # Namespace resize
+              args:
+                subsystem: nqn.2016-06.io.spdk:cnode1.gw-group1
+                nsid: 2
+                size: 15G
+              validate:
+                omap: true
+          - config:
+              service: namespace
+              command: change_load_balancing_group              # Namespace change_load_balancing_group
+              args:
+                subsystem: nqn.2016-06.io.spdk:cnode1.gw-group1
+                nsid: 3
+                load-balancing-group: 2
+              validate:
+                omap: true
+          - config:
+              service: namespace
+              command: change_load_balancing_group              # Namespace change_load_balancing_group
+              args:
+                subsystem: nqn.2016-06.io.spdk:cnode1.gw-group1
+                nsid: 4
+                load-balancing-group: 1
+              validate:
+                omap: true
+          - config:
+              service: namespace
+              command: list             # namespace list
+              args:
+                subsystem: nqn.2016-06.io.spdk:cnode1.gw-group1
+              base_cmd_args:
+                format: json
+          - config:
+              service: namespace
+              command: get_io_stats             # namespace get_io_stats
+              args:
+                subsystem: nqn.2016-06.io.spdk:cnode1.gw-group1
+                nsid: 1
+          - config:
+              service: namespace
+              command: del_host             # namespace del_host
+              args:
+                subsystem: nqn.2016-06.io.spdk:cnode1.gw-group1
+                nsid: 3
+                host: nqn.2014-08.org.nvmexpress:uuid:7c1c2d62-2ad0-4181-ae79-8e780a6a080b
+              validate:
+                omap: true
+          - config:
+              service: namespace
+              command: del_host             # namespace del_host
+              args:
+                subsystem: nqn.2016-06.io.spdk:cnode1.gw-group1
+                nsid: 4
+                host: nqn.2014-08.org.nvmexpress:uuid:7c1c2d62-2ad0-4181-ae79-8e780a6a080c
+              validate:
+                omap: true
+          - config:
+              service: namespace
+              command: change_visibility              # Namespace change_visibility
+              args:
+                subsystem: nqn.2016-06.io.spdk:cnode1.gw-group1
+                nsid: 3
+                auto-visible: true
+              validate:
+                omap: true
+          - config:
+              service: namespace
+              command: change_visibility              # Namespace change_visibility
```

Comment:
@rahullepakshi 
lib method is missing for the new functionalities. Please check

Reply:
sorry, its present. Please ignore.